package.json

```diff
@@ -23,7 +23,8 @@
     "precommit": "lint-staged",
     "prettier": "yarn format:source && yarn format:examples",
     "prettier:diff": "yarn nit:source && yarn nit:examples",
-    "test": "jest"
+    "test": "jest",
+    "start": "cd website && yarn start"
```

Comment:
😆

Reply:
I was too tired of switching directories. babel-website does this too.